servers/zts/schema/zts_server.sql

```diff
@@ -35,6 +35,7 @@ CREATE TABLE IF NOT EXISTS `zts_store`.`certificates` (
   `lastNotifiedServer` VARCHAR(512) NULL,
   `expiryTime` DATETIME(3) NULL,
   `hostName` VARCHAR(512) NULL,
+  `siaProvider` VARCHAR(256) NOT NULL,
```

Comment:
These are the instructions for the ZMS schema but they apply to ZTS as well:

https://github.com/AthenZ/athenz/blob/master/servers/zms/schema/schema_update.md